__tests__/reviews.test.js

```diff
@@ -0,0 +1,38 @@
+const request = require("supertest");
+const seed = require('../db/seeds/seed');
+const db = require('../db/connection');
+const app = require("../app");
+const testdata = require("../db/data/test-data/");
+
+beforeEach(() => {
+  return seed(testdata);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe('200 - GET /api/reviews', () => {
+test('should return an array of review objects', () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({body : {reviews}}) => {
+        reviews.forEach((review) => {
+          expect(Object.keys(review).length).toBe(9);
```

Comment:
This assertion makes this test too brittle - the reason we encourage you to use toMatchObject rather than toEqual is to allow for extensibility of adding more properties to objects in future - by asserting the exact amount of properties this test will break if future changes are made - the rest of this tests looks great but get rid of line 22 here 👍

Reply:
Same for /api/categories 👍